### PR TITLE
Fix demo delivery report and delivery report orientation

### PR DESCRIPTION
**Pull request by northwestwitch**
Show a partial fix to #3108 

- Fixes missing delivery report for demo case in Scout
- Replaces delivery report file with a more recent file with additional cols and landscape iroentation 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh your.email@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. Locally: on master branch, make sure the delivery report doesn't work.
2. Switch to this branch and make sure that delivery report works and it's exported in PDF format with all the text fitting the page (page doesn't have a logo)
3. Install on cg-vm1 and check that delivery reports are shown for existing cases (PDF report might not be fitting the page)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/demo/delivery_report.html [hunk 1]
```diff
@@ -14,12 +13,11 @@
...
-  </style>
-
+</style>
 
-  <style>
+<style>
     @page {
-      size: A4;
```
Comment (northwestwitch) [inline scout/demo/delivery_report.html]: This is the actual fix for the content not fitting the PDF reports in #3108

**Comment by northwestwitch** [comment]
Locally, 
- After fixing missing report for demo case
- New report (with additional cols) with **A4 orientation**  (as in current prod)
- Running scout from docker-compose: ⛔ 

image

Doing the same but with landscape orientation of the report:  content fits the page 🆗 

image

File: scout/server/blueprints/cases/views.py [hunk 2]
```diff
@@ -680,6 +680,9 @@ def delivery_report(institute_id, case_name):
     else:
         delivery_report = case_obj["delivery_report"]
 
+    out_dir = os.path.abspath(os.path.dirname(delivery_report))
```
Comment (dnil) [inline scout/server/blueprints/cases/views.py:683]: 🤷

File: scout/demo/delivery_report.html (hunk 1 above)
Comment (dnil) [reply scout/demo/delivery_report.html:22]: 🥇

**Review by dnil — APPROVED** [review]
Nice to keep it current! I noticed vulture had some suggestions; if you have time, perhaps check and fix, or mark them noqa if they are incorrect, when you are anyway changing the file.

File: scout/server/blueprints/cases/views.py (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/cases/views.py:683]: yeah, that fixes the demo delivery report not found

**Comment by northwestwitch** [comment]
I'm going to install it on cg-vm1 before merging, but I'm afraid we don't have many cases with delivery report in there..

**Comment by northwestwitch** [comment]
Ok, found a case with delivery report that looks ugly on cg-vm1: https://scout-stage.scilifelab.se/cust002/F0042222

Installing this branch on that server doesn't fix the error, because the HTML report should be fixed instead., but the download works (so the PR is not breaking the functionality):

image

After manually fixing the HTML report (changing the report orientation to landscape):

image

The PDF report looks fine: 🆗 

image

**Pull request merged**